src/aks-preview/HISTORY.rst

```diff
@@ -12,6 +12,10 @@ To release a new version, please select a new version number (usually plus 1 to
 Pending
 +++++++
 
+0.5.123
++++++++
+* Update command group `az aks maintenanceconfiguration` to enable the creation of *aksManagedAutoUpgradeSchedule* and *aksManagedNodeOSUpgradeSchedule*. 
```

Comment:
> *aksManagedAutoUpgradeSchedule* and *aksManagedNodeOSUpgradeSchedule*

Could we describe these two features in detail in the history notes, otherwise their meaning is easily confusing?

Reply:
Thanks for your feedback, I have updated the details!